### PR TITLE
Dev/white balance

### PR DESCRIPTION
**Pull request by servantf**
Add an option to ignore the white balance post processing in LibRaw. 

CameraInit will set this parameter in the SfmData::Views given command line parameters

Option is passed to image reading function.

For the moment, only using this option in panorama related nodes

### CODE REVIEW

File: src/aliceVision/hdr/sampling.cpp [hunk 1]
```diff
@@ -169,8 +169,12 @@ bool Sampling::extractSamplesFromImages(std::vector<ImageSample>& out_samples, c
     {
         const float exposure = times[idBracket];
 
+        image::SImageReadOptions options;
+        options.outputColorSpace = image::EImageColorSpace::LINEAR;
```
Comment (fabiencastan) [inline src/aliceVision/hdr/sampling.cpp:173]: should use `colorspace` param

File: src/software/pipeline/main_LdrToHdrMerge.cpp [hunk 2]
```diff
@@ -277,7 +277,11 @@ int aliceVision_main(int argc, char** argv)
         {
             const std::string filepath = group[i]->getImagePath();
             ALICEVISION_LOG_INFO("Load " << filepath);
-            image::readImage(filepath, images[i], image::EImageColorSpace::SRGB);
+
+            image::SImageReadOptions options;
+            options.outputColorSpace = image::EImageColorSpace::LINEAR;
```
Comment (fabiencastan) [inline src/software/pipeline/main_LdrToHdrMerge.cpp:282]: Colorspace param was `image::EImageColorSpace::SRGB`.

File: src/aliceVision/image/io.hpp [hunk 3]
```diff
@@ -42,6 +42,20 @@ enum class EImageFileType
   EXR
 };
 
+/**
+ * @brief aggregate for multiple image reading options
+ */
+struct SImageReadOptions
```
Comment (fabiencastan) [inline src/aliceVision/image/io.hpp:48]: I would remove the S prefix as we are not using it for any other struct.
```suggestion
struct ImageReadOptions
```

File: src/software/pipeline/main_LdrToHdrMerge.cpp (hunk 2 above)
Comment (servantf) [reply src/software/pipeline/main_LdrToHdrMerge.cpp:282]: oups

File: src/aliceVision/hdr/sampling.cpp (hunk 1 above)
Comment (servantf) [reply src/aliceVision/hdr/sampling.cpp:173]: oups !!! did it too early in the morning

File: src/aliceVision/image/io.hpp [hunk 4]
```diff
@@ -45,7 +45,7 @@ enum class EImageFileType
 /**
  * @brief aggregate for multiple image reading options
  */
-struct SImageReadOptions
+struct ImageReadOptions
```
Comment (fabiencastan) [inline src/aliceVision/image/io.hpp:48]: Need to do it from a refactoring tool.

**Review by fabiencastan — APPROVED** [review]

**Pull request merged**